Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:10.19.0
+LABEL author="ODK"
+LABEL maintainer="ODK Build maintainers"
+LABEL description="ODK build2xlsform, an ODK XForms to XLSForm converter"
```

Comment:
This is misleading because it makes it seem like any XForm can be converted to an XLSForm. Maybe "Generate an XLSForm from an ODK Build form" or something like that.

Reply:
Pushed "Generate an XLSForm from an ODK Build Xform" as a suggestion of "XForm" over "form". How does that read?